docs/API.md

```diff
@@ -106,3 +108,67 @@ resolvers = {
 }
 ```
 4. **directive_resolvers:** Resolver list which is used as the directive resolvers.
+
+## Resolver
+
+### Declare resolvers
+
+Tartiflette provides 2 different ways to declare a resolver for a Field.
+
+#### With decorators
+
+The most common way to asign a specific resolver for a Field is to put a decorator on top of the resolver function, that one [MUST BE compliant with the function signature](#function-signature).
+
+```python
+from tartiflette import Resolver
+
+@Resolver("Query.hello")
+def my_hello_resolver(parent, args, context, info):
+    return "Chuck"
+```
+
+#### During Engine initialization
+
+The second way is to use the [advanced Engine initialization](#advanced-constructor) to specify a list of resolvers associated to Fields.
+
+```python
+import tartiflette
+
+def my_hello_resolver(parent, args, ctx, info):
+    return "Chuck"
+
+engine = tartiflette.Engine(
+    sdl,
+    resolvers={
+        "Query.hello": my_hello_resolver
+    }
+)
+```
+
+### Function signature
+
+Every resolver in Tartiflette accepts four positional arguments:
+
+_(This signature has been highly inspired by the GraphQL.js implementation)_
+
+```python
+def my_hello_resolver(parent, args, context, info):
+    pass
+```
+
+1. **parent:** Object which contains the result returned by the resolver of the parent field. The `root_value` is passed in the case of a top-level Query field.
```

Comment:
Object which contains ? What do you mean ? A class ? An instance ? 
Maybe just write "The result returned by the resolver of the parent field".